src/server.js

```diff
@@ -317,12 +317,17 @@ app.use('/requestPasswordReset', (req, res) => {
 
 function orientImageBuffer({ attachmentBuffer }) {
   console.time(`orientImageBuffer`); // eslint-disable-line no-console
+  // eslint-disable-next-line no-console
+  console.log(
+    `image buffer length BEFORE sharp: ${attachmentBuffer.length} bytes`,
+  );
   return sharp(attachmentBuffer)
     .rotate()
     .toBuffer()
     .catch(() => attachmentBuffer)
     .then(buffer => Buffer.from(buffer))
     .then(buffer => {
+      console.log(`image buffer length AFTER sharp: ${buffer.length} bytes`); // eslint-disable-line no-console
```

Comment:
> plate recognizer has a file size limit of 3MB, and recommends an image resolution of 1024x768, so we'll probably need to use e.g. `sharp` to resize images to make them under 3MB: [app.platerecognizer.com/upload-limit](https://app.platerecognizer.com/upload-limit/)
> 
> ![image](https://user-images.githubusercontent.com/6473925/176568744-d40ffe9f-7444-4fe3-9592-47eecfc2f5a3.png)

> See [sharp.pixelplumbing.com/api-resize](https://sharp.pixelplumbing.com/api-resize) for documentation on how to do this with sharp

It seems like sharp automatically makes images smaller. I tried a 3.7MB image, and here's what printed out:

```
image buffer length BEFORE sharp: 3723317 bytes
image buffer length AFTER sharp: 1962551 bytes
orientImageBuffer: 391.626ms
STARTING platerecognizer
/platerecognizer plate-reader: 25.706s
```

so... maybe we don't need to explicitly resize the image after all?